app/renderer/services/get_chrome_versions.bat

```diff
@@ -0,0 +1,27 @@
+@ECHO OFF
+
+:: Look for machine-wide Chrome installs (stable, Beta, and Dev).
+:: Get the name, running version (if an update is pending relaunch), and
+:: installed version of each.
+FOR %%A IN (
+    {8A69D345-D564-463c-AFF1-A69D9E530F96},
+    {8237E44A-0054-442C-B6B6-EA0509993955},
+    {401C381F-E0DE-4B85-8BD8-3F3F14FBDA57}) DO (
+  reg query HKLM\Software\Google\Update\Clients\%%A /v name /reg:32 2> NUL
+  reg query HKLM\Software\Google\Update\Clients\%%A /v opv /reg:32 2> NUL
+  reg query HKLM\Software\Google\Update\Clients\%%A /v pv /reg:32 2> NUL
+)
+
+:: Look for Chrome installs in the current user's %LOCALAPPDATA% directory
+:: (stable, Beta, Dev, and canary).
+:: Get the name, running version (if an update is pending relaunch), and
+:: installed version of each.
+FOR %%A IN (
+    {8A69D345-D564-463c-AFF1-A69D9E530F96},
+    {8237E44A-0054-442C-B6B6-EA0509993955},
+    {401C381F-E0DE-4B85-8BD8-3F3F14FBDA57},
+    {4ea16ac7-fd5a-47c3-875b-dbf4a2008c20}) DO (
+  reg query HKCU\Software\Google\Update\Clients\%%A /v name /reg:32 2> NUL
+  reg query HKCU\Software\Google\Update\Clients\%%A /v opv /reg:32 2> NUL
+  reg query HKCU\Software\Google\Update\Clients\%%A /v pv /reg:32 2> NUL
+)
```

Comment:
I assume this file is not needed since the one from app/main/services/MenuService/get_chrome_versions.bat  is used?

Reply:
fixed https://github.com/oxygenhq/oxygen-ide/pull/44/commits/27047afd6704c1f74e90a6c5b7470a89cc7a64b1